src/modules/lists/interfaces/updatelist.interface.ts

```diff
@@ -0,0 +1,6 @@
+import { Document } from 'mongoose';
+
+export interface UpdateList extends Document {
```

Comment:
I don't think this is necessary, we already have an interface for `List`, please remove this file